src/App.tsx

```diff
@@ -1,61 +1,94 @@
+import { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+
+export const App: React.FC = () => {
+  const [currentTitle, setCurrentTitle] = useState('');
+  const [currentUserId, setCurrentUserId] = useState(0);
+  const [todos, setTodos] = useState(todosFromServer);
+  const [wasSubmitted, setWasSubmitted] = useState(false);
+
+  const isFormValid = currentTitle.trim() && currentUserId;
```

Comment:
The `isFormValid` logic could be improved for readability. Consider using explicit boolean expressions, such as `currentTitle.trim() !== '' && currentUserId !== 0`, to make the condition clearer.